### PR TITLE
LA-175 - Adding "Ondsel Start" tab

### PR DESCRIPTION
**Pull request by JohnAD**
https://ondsel-brad.atlassian.net/browse/LA-175?atlOrigin=eyJpIjoiYmUwYmVmMmI1MDQyNDFhNjhhODc0ZTIzNDhhYzlkMWIiLCJwIjoiaiJ9

### CODE REVIEW

File: markdown.py
Comment (pieterhijma) [inline markdown.py:1]: Isn't it better to use a library for this? I see three candidates `markdown`, `mistune`, and `markdown2`.

**Review by pieterhijma — CHANGES_REQUESTED** [review]
Looks good with some minor comments. I'm also ok with creating cards to fix the issues later.

File: WorkspaceView.py [hunk 1]
```diff
@@ -4,6 +4,14 @@
 # *                                                                     *
 # ***********************************************************************
 
+# # TODO: the following try/except is a hack that only works while online for dev-testing
```
Comment (JohnAD) [inline WorkspaceView.py:7]: To test, you might need to uncomment this code to import the new library. Actual add-on users will not have a problem as the Add-On Manager will honor the updates `package.xml` file.

Comment (pieterhijma) [reply WorkspaceView.py:7]: I've added a commit that simply imports `mistune`.

In the meantime you can install `mistune` in the following way: 

``` python
import Addon
from addonmanager_dependency_installer import DependencyInstaller

addon = Addon.Addon(
    name="Ondsel-Lens", 
    url="https://github.com/Ondsel-Development/Ondsel-Lens-Addon",
    branch="feature/LA-175-ondsel-start-tab",
)
dep_installer = DependencyInstaller([addon], ["mistune"], [])
dep_installer.run()
```
 
 Note that the branch may need to be `develop` after this PR has merged.

**Review by pieterhijma — APPROVED** [review]
Pushed some minor changes, but looks good!

**Pull request merged**